torch/_prims/nvfuser_prims.py

```diff
@@ -391,16 +391,26 @@ def _dim_size(a, dims):
     return reduction_size
 
 
+def _restore_reduced_dims(a, dims, shape):
+    unsqueezed_a = _unsqueeze_dims(a, dims, len(shape))
+    return _expand(a, shape)
+
+
+# Reference: https://github.com/pytorch/pytorch/blob/master/tools/autograd/derivatives.yaml#L1109-L1115
+def _amax_amin_vjp(grad, result, self, dims, keepdim):
+    expanded_grad = _restore_reduced_dims(grad, dims, self.shape)
+    expanded_result = _restore_reduced_dims(result, dims, self.shape)
+    mask = torch.eq(expanded_result, self)
+    return (expanded_grad / torch.sum(mask, dims, keepdim=True)) * mask
```

Comment:
Interface in the current iteration expects 
```suggestion
    return (expanded_grad / torch.sum(mask, dims, keepdim=True)) * mask, *(None,) * (len(dims) + 1)
```
One None for each dim and one None for keepdim argument.